### PR TITLE
:rocket::rocket:

### PR DESCRIPTION
**Pull request by sethkrasnianski**
@lupinskij :eyes:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "url": "git@github.com:schwill/boilerplate.git"
+  },
+  "scripts": {
+    "start": "gulp",
+    "build": "NODE_ENV=production gulp build",
+    "clean": "rm -rf public"
+  },
+  "devDependencies": {
+    "babel": "^5.8.23",
+    "babelify": "^6.2.0",
```
Comment (lupinskij) [inline package.json]: Failed to load external module babel-core/register

File: plug.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (lupinskij) [inline plug.json]: Thoughts on naming this file gulp-config.json?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "type": "git",
+    "url": "git@github.com:schwill/boilerplate.git"
+  },
+  "scripts": {
+    "start": "gulp",
+    "build": "NODE_ENV=production gulp build",
+    "clean": "rm -rf public"
+  },
+  "devDependencies": {
+    "babel": "^5.8.23",
```
Comment (lupinskij) [inline package.json]: Suggest using babel-core, otherwise returns Failed to load external module babel-core/register

File: gulpfile.babel.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+      locals: locals
+    }))
+    .pipe(gulp.dest(config.build))
+    .pipe($.if(!production, $.notify({
+      title: 'Sucess',
+      message: 'Views compiled'
+    })));
+});
+
+// Templates
```
Comment (lupinskij) [inline gulpfile.babel.js]: Only comment?

Comment (sethkrasnianski) [inline gulpfile.babel.js]: Will remove

File: package.json (hunk 3 above)
Comment (sethkrasnianski) [inline package.json]: Will add

File: plug.json (hunk 2 above)
Comment (sethkrasnianski) [inline plug.json]: fine :cry:

**Comment by lupinskij** [comment]
:cake:

**Pull request merged**